core/arch/riscv/kernel/trace_ext.c

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
This file looks identical to the arch/arm version of the same. We should move `core/arch/arm/kernel/trace_ext.c` to `core/kernel/trace_ext.c` instead.